apps/aehttpclient/lib/client.ex

```diff
@@ -41,10 +49,12 @@ defmodule Aehttpclient.Client do
           :info ->
             response = Poison.decode!(body, keys: :atoms!)
             {:ok, response}
-          :peers ->
```

Comment:
why is `:peers` and `:balance` removed?

Reply:
:peers and :balance will be handled by the default json_response()

Reply:
great